spec/configuration/sugar_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'spec_helper'
+
+module Xcake
+  describe Configurable do
+
+    before :each do
+      @configuration = Configuration.new('test')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.